idea/crateApp.xml

```diff
@@ -0,0 +1,29 @@
+<configuration default="false" name="Crate" type="Application" factoryName="Application">
+  <extension name="coverage" enabled="false" merge="false" />
+  <option name="MAIN_CLASS_NAME" value="io.crate.bootstrap.CrateF" />
+  <option name="VM_PARAMETERS" value="-Xmx512m -Xms512m -Des.path.home=$PROJECT_DIR$/sandbox/crate" />
```

Comment:
do we really want to set HEAP size here? I think its better to rely on JVM's defaults by default